packages/material-ui/src/CardMedia/CardMedia.js

```diff
@@ -14,6 +14,8 @@ export const styles = {
   /* Styles applied to the root element if `component="video, audio, picture, iframe, or img"`. */
   media: {
     width: '100%',
+  },
+  img: {
```

Comment:
Can you add a description?

Something along the lines of:

Styles applied to the root element if `component="picture"` or `component="img"`.